1-mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,71 @@
 // Write your code here
+const images = ["url('./assets/img1.jpg')", "url('./assets/img2.jpg')", "url('./assets/img3.jpg')", "url('./assets/img4.jpg')", "url('./assets/img5.jpg')"]
```

Comment:
not sure where these are, but always try to make sure they are not stored on your computer, as I can't see them at all! also would have been good to see alt tags for these so that people like me who can't see then know what the image is of!